packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -153,6 +166,19 @@ class LocalStoreTester {
     return this;
   }
 
+  afterBundleDocuments(documents: BundledDocuments): LocalStoreTester {
+    this.prepareNextStep();
+
+    this.promiseChain = this.promiseChain
+      .then(() => {
+        return this.localStore.applyBundledDocuments(documents);
```

Comment:
Note that you used both `afterBundleDocuments` and `applyBundledDocuments` here.

Reply:
You were right!